### DISCUSSION TITLE
SVG work in Spoke but not in Hubs?

### DISCUSSION THREAD

**Discussion by github-user-77** [mozilla/hubs#3310] [Spoke Questions] [answered]
SVGs seem to load into Spoke and are displayed perfectly, but the scene in Hubs Cloud shows a broken link image. SVG address confirmed in browser and visible in Spoke. Does Hubs not support SVG but Spoke does?

**Comment by pattersonbl2** [comment, marked as answer]
I have opened a issue request for this problem : https://github.com/mozilla/hubs/issues/4493
